python/database.py

```diff
@@ -0,0 +1,172 @@
+import logging
+
+from sqlalchemy import create_engine, Column, Integer, String, DateTime
+from sqlalchemy.orm import sessionmaker, declarative_base
+from datetime import datetime
+from sqlalchemy.exc import IntegrityError
+
+log = logging.getLogger(__name__)
+handler = logging.StreamHandler()  # Logs to the terminal
+log.addHandler(handler)  # Adds handler to log
+
+# Base class for ORM models
+Base = declarative_base()
+
+class User(Base):
+    __tablename__ = 'users'
+    
+    id = Column(Integer, primary_key=True)
+    username = Column(String, nullable=False)
+    email = Column(String, unique=True, nullable=False)
+    password_hash = Column(String, nullable=False)
+    created_at = Column(DateTime, default=datetime.utcnow)
```

Comment:
```suggestion
    created_at = Column(DateTime, default=datetime.now(timezone.utc))
```
`.utcnow` is deprecated 
![image](https://github.com/user-attachments/assets/ffa628ad-17da-4aba-a0a4-e2adf636e88f)


edit: implemented in #9